### ISSUE TITLE
[GLPI 10.0.12] - I can no longer add tickets

### ISSUE THREAD

**Issue by RobertoMarcelino** [glpi-project/glpi#16535]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.12

### Bug description

After upgrading from version 10.0.11 to 10.0.12, I can no longer add tickets.

``

### Relevant log output

```shell
[2024-02-08 14:57:25] glpisqllog.ERROR: DBmysql::doQuery() in /var/www/html/glpi/src/DBmysql.php line 403
  *** MySQL query error:
  SQL: INSERT INTO `glpi_tickets` (`name`, `content`, `entities_id`, `date`, `type`, `itilcategories_id`, `status`, `requesttypes_id`, `urgency`, `impact`, `priority`, `locations_id`, `actiontime`, `slas_id_tto`, `slas_id_ttr`, `olas_id_tto`, `olas_id_ttr`, `users_id_lastupdater`, `users_id_recipient`, `global_validation`, `slalevels_id_ttr`, `sla_waiting_duration`, `time_to_resolve`, `olalevels_id_ttr`, `ola_ttr_begin_date`, `ola_waiting_duration`, `internal_time_to_resolve`, `time_to_own`, `ola_tto_begin_date`, `internal_time_to_own`, `date_creation`, `date_mod`) VALUES ('Desligar servidores', '&#60;p&#62;Desligar servidores.&#60;/p&#62;', '11', '2024-02-08 14:57:24', '2', '408', '1', '2', '3', '3', '3', '13', '0', '3', '4', '3', '4', '6', '6', '1', '7', '', '2024-02-12 14:57:24', '0', '2024-02-08 14:57:24', '', '2024-02-12 14:57:24', '2024-02-08 18:57:24', '2024-02-08 14:57:24', '2024-02-08 18:57:24', '2024-02-08 14:57:24', '2024-02-08 14:57:24')
  Error: Incorrect integer value: '' for column `glpi`.`glpi_tickets`.`sla_waiting_duration` at row 1
  Backtrace :
  src/DBmysql.php:1380                               DBmysql->doQuery()
  src/CommonDBTM.php:729                             DBmysql->insert()
  src/CommonDBTM.php:1342                            CommonDBTM->addToDB()
  front/ticket.form.php:80                           CommonDBTM->add()
  public/index.php:82                                require()
  {"user":"6@xxxxxx"}

[2024-02-08 15:00:50] glpisqllog.ERROR: DBmysql::doQuery() in /var/www/html/glpi/src/DBmysql.php line 403
  *** MySQL query error:
  SQL: INSERT INTO `glpi_tickets` (`name`, `content`, `entities_id`, `date`, `type`, `itilcategories_id`, `status`, `requesttypes_id`, `urgency`, `impact`, `priority`, `locations_id`, `actiontime`, `slas_id_tto`, `slas_id_ttr`, `olas_id_tto`, `olas_id_ttr`, `users_id_lastupdater`, `users_id_recipient`, `global_validation`, `slalevels_id_ttr`, `sla_waiting_duration`, `time_to_resolve`, `olalevels_id_ttr`, `ola_ttr_begin_date`, `ola_waiting_duration`, `internal_time_to_resolve`, `time_to_own`, `ola_tto_begin_date`, `internal_time_to_own`, `date_creation`, `date_mod`) VALUES ('Desligar servidores', '&#60;p&#62;Desligar servidores.&#60;/p&#62;', '11', '2024-02-08 15:00:50', '2', '408', '1', '2', '3', '3', '3', '13', '0', '3', '4', '3', '4', '6', '6', '1', '7', '', '2024-02-12 15:00:50', '0', '2024-02-08 15:00:50', '', '2024-02-12 15:00:50', '2024-02-08 19:00:50', '2024-02-08 15:00:50', '2024-02-08 19:00:50', '2024-02-08 15:00:50', '2024-02-08 15:00:50')
  Error: Incorrect integer value: '' for column `glpi`.`glpi_tickets`.`sla_waiting_duration` at row 1
  Backtrace :
  src/DBmysql.php:1380                               DBmysql->doQuery()
  src/CommonDBTM.php:729                             DBmysql->insert()
  src/CommonDBTM.php:1342                            CommonDBTM->addToDB()
  front/ticket.form.php:80                           CommonDBTM->add()
  public/index.php:82                                require()
  {"user":"6@xxxxxx"}
```

**Comment by cconard96** [comment]
Hello,
This issue should be fixed by #16496

**Comment by RobertoMarcelino** [comment]
Thank you very much.
